### PR TITLE
Change delete file modal confirmation process

### LINKED ISSUE #3163: Change delete file modal confirmation process
**From user testing:**

Currently, to delete a file, you must confirm the entire file name:

screenshot from 2018-03-07 20-44-40

This is a little bit painful for some users, because the file name can get pretty long.

Instead, we should change this modal so that we require the `project name` for confirmation.

---

**Good First Issue**: This issue is good for first time contributors. If you've already contributed to Warehouse, please work on [another issue without this label](https://github.com/pypa/warehouse/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+-label%3A%22good+first+issue%22) instead. If there is not a corresponding pull request for this issue, it is up for grabs. For directions for getting set up, see our [Getting Started Guide](https://warehouse.pypa.io/development/getting-started/). If you are working on this issue and have questions, please feel free to ask them here, [`#pypa-dev` on Freenode](https://webchat.freenode.net/?channels=%23pypa-dev), or the [pypa-dev mailing list](https://groups.google.com/forum/#!forum/pypa-dev).

### PR DESCRIPTION
**Pull request by aalmazan**
Fixes #3163. 

* Adds an additional optional parameter for the `confirm_modal` macro which will display after the warning box.
* Project name will be used instead of filename to confirm delete.

### CODE REVIEW

**Comment by di** [comment]
Specifically this view: https://github.com/pypa/warehouse/blob/a6d34187a91d4a7f872079d04d26c19eacbcec48/warehouse/manage/views.py#L450-L512

**Comment by aalmazan** [comment]
@di Hmm, just noticed that this is still marked as "Changes Requested". I have the requested changes in 7e6b4e28ed. Is there maybe something missing on my end to change the review status?

**Comment by nlhkabu** [comment]
hi @aalmazan - I think we just need @di to 'approve' the PR to update the status.  He is working on another project this week, so this might take more time than usual :)

File: warehouse/templates/manage/release.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h2 class="heading-wsubtitle__heading">Release Version {{ release.version }}</h2
                 <i class="fa fa-hashtag" aria-hidden="true"></i>
                 View Hashes
               </a>
-              {% set title = "Delete File From" %}
+              {% set title = "Delete File" %}
```
Comment (di) [inline warehouse/templates/manage/release.html]: This makes the title of the modal say "Delete File {project name}":

screen shot 2018-04-02 at 3 35 19 pm

I think we need to find a way to have differing title and button texts.

Comment (nlhkabu) [reply warehouse/templates/manage/release.html]: To be explicit here: We want the title to be `Delete file from {project name}` and the button to be `Delete file`

**Comment by aalmazan** [comment]
Updated the modal with the requested changes. This adds another parameter to `confirm_modal`: `confirm_button_label`. If that doesn't exist, `title` will be used instead (as it did before this change).

Example below: 
selection_035

**Pull request merged**